sklearn/tests/test_calibration.py

```diff
@@ -547,6 +547,19 @@ def test_calibration_pipeline(text_data, text_data_pipeline):
         calib_clf.n_features_in_
 
 
+def test_calibration_pipeline_prediction(text_data, text_data_pipeline):
+    # Test that calibration works in prefit pipeline with transformer,
+    # where `X` is not array-like, sparse matrix or dataframe at the start.
+    # Test is can also predict without running into validation errors.
+    # See https://github.com/scikit-learn/scikit-learn/issues/19637
```

Comment:
Trying to move toward docstrings for test so that we can validate that all tests have a comment:

```suggestion
    """Test that calibration works in prefit pipeline with transformer,
    where `X` is not array-like, sparse matrix or dataframe at the start.
    Test is can also predict without running into validation errors.
    See https://github.com/scikit-learn/scikit-learn/issues/19637"""
```

Reply:
Thank you for your feedback! I'm quite new to this. I will make these changes now.